roles/common/tasks/main.yml

```diff
@@ -62,6 +62,11 @@
   tags:
     - validate
 
+- name: Symlink bash on z/OS
```

Comment:
Can we please use ansible file or some other core module for the this.

Reply:
I wanted to, originally, but built in module was failing, complaining /bin/bash didn't exist.